testing-mysql-server-5/src/main/java/com/facebook/presto/testing/mysql/EmbeddedMySql5.java

```diff
@@ -30,31 +30,60 @@ public EmbeddedMySql5(MySqlOptions mySqlOptions)
     @Override
     public List<String> getInitializationArguments()
     {
-        return ImmutableList.of(
+        if (isMariadb) {
+            return ImmutableList.of(
                 "--no-defaults",
-                "--initialize-insecure",
+                "--basedir=" + getBaseDirectory(),
```

Comment:
the difference here is that mariadb also gets a base dir arg?  Can you put the arguments in the same order so it's easier to tell?  maybe even  build the common elements of the list first and then add the mariadb special properties.

Reply:
Right, I'll do that.